lib/babe/mocks_generate_test.go

```diff
@@ -0,0 +1,6 @@
+// Copyright 2022 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package babe
+
+//go:generate mockgen -destination=mocks/network.go -source=../../dot/core/interface.go -package mocks dot/core Network
```

Comment:
@qdm12 Would you know whats wrong here? it mocks all interfaces instead of just the network interface

Reply:
Don't use the `-source` mode (also don't generate to a mocks directory, generate to a local _test file):

```go
//go:generate mockgen -destination=mocks_test.go -package=$GOPACKAGE github.com/ChainSafe/gossamer/dot/core 
 Network
```

Reply:
done.

> also don't generate to a mocks directory, generate to a local _test file
There is already a mocks directory here. I would prefer to just let it stay the way it is.